client/my-sites/stats/controller.jsx

```diff
@@ -143,7 +140,7 @@ export function overview( context, next ) {
 	const siteId = getSelectedSiteId( context.store.getState() );
 
 	context.primary =
-		isPurchaseFlowEnabled && siteId !== null ? (
+		siteId !== null ? (
```

Comment:
Why do we need to keep the other part of the condition (lines 156:161)?

Reply:
if no site is specified, it shows an overview of all sites, which is a feature I believe.

![image](https://github.com/Automattic/wp-calypso/assets/1425433/e5030c11-9e55-40ea-ae57-9b8315b44042)

Reply:
It points to the same component with the same properties, so I think that the check should have been applied to both cases. This would mean that the second part of the condition should have been removed with the feature flag.

Reply:
Thanks for the catch @grzegorz-cp. I'll do some additional testing around this before merging.